opsdroid/connector/slack/events.py

```diff
@@ -125,9 +129,26 @@ def __init__(self, payload, *args, **kwargs):
 class ViewClosed(InteractiveAction):
     """Event class to represent view_closed in Slack."""
 
-    def __init__(self, payload, *args, **kwargs):
-        """Create object with minimum properties."""
-        super().__init__(payload, *args, **kwargs)
+
+class SlashCommand(InteractiveAction):
+    """
+    Event class to represent a slash command.
+
+    args:
+        payload: Incomming payload from the Slack API
+
+    **Basic Usage in a Skill:**
+
+    .. code-block:: python
+
+        from opsdroid.skill import Skill
+        from opsdroid.matchers import match_regex
+
+        class SearchMessagesSkill(Skill):
+            @match_event(SlashCommand, command="/testcommand")
+            async def slash_command(self, message):
+                await message.respond("Hi")
```

Comment:
I think the main change I would like to see here is to have an example of accessing some arguments passed to the command via the payload.

Reply:
This is resolved